corehq/apps/es/README.rst

```diff
@@ -402,12 +402,6 @@ Client adapters are split into two usage patterns, the "Management Adapter" and
 to perform index verification when Django starts.  Downstream code needing an
 adapter import and use the adapter instance.
 
-.. toctree::
-
-    Management Adapter
-    Document Adapters
-    Code Documentation
-
```

Comment:
Memory does not serve me well on this one. I like your solution (remove the toctree). Besides, the side panel tree seems to work fine for whatever this might have achieved at one time:

![image](https://user-images.githubusercontent.com/1731363/221653644-63d12682-46cf-441d-9584-3dca01d396f8.png)